cli/cdsctl/workflow_as_code.go

```diff
@@ -0,0 +1,342 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	repo "github.com/fsamin/go-repo"
+
+	"github.com/ovh/cds/cli"
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/cdsclient"
+	"github.com/ovh/cds/sdk/exportentities"
+)
+
+var workflowInitCmd = cli.Command{
+	Name:  "init",
+	Short: "Init a workflow",
+	Long: `
+		Initialize a workflow from your current repository..
```

Comment:
```space will create <pre> in doc```

Initialize a workflow from your current repository, this will create yml files and push them to CDS.